packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -43,8 +81,191 @@ const ReleasesPage = () => {
           </CheckPermissions>
         }
       />
+      {children}
+    </Main>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * ReleasesGrid
+ * -----------------------------------------------------------------------------------------------*/
+interface ReleasesGridProps {
+  sectionTitle: 'pending' | 'done';
+  releases?: GetReleases.Response['data'];
+  isError?: boolean;
+}
+
+const LinkCard = styled(Link)`
+  display: block;
+`;
+
+const ReleasesGrid = ({ sectionTitle, releases = [], isError = false }: ReleasesGridProps) => {
+  const { formatMessage } = useIntl();
+
+  if (isError) {
+    return (
+      <Flex paddingTop={8}>
+        <AnErrorOccurred />
+      </Flex>
+    );
+  }
```

Comment:
Just want to clarify this gives an error message in the tab right (not an error page)? Why do we need the paddingTop if we have the paddingBottom from the tabs?